src/Data/Audience.elm

```diff
@@ -36,6 +40,45 @@ type alias Audience =
 
 
 
+-- Decoders
+
+
+decoder : Decoder Audience
+decoder =
+    -- I know about Json.Decode.Pipeline but for small projects, I prefer to
+    -- limit the number of dependencies. Plus, using `mapX` leads to easier
+    -- to understand error messages than the applicative version.
+    Json.Decode.map4 Audience
+        (Json.Decode.field "id" Json.Decode.int)
+        (Json.Decode.field "name" Json.Decode.string)
+        -- I choosed to rely on the `type` since it seems way easier
+        -- (and more reliable) than checking `curated` and `shared`
+        -- booleans
```

Comment:
:+1: Makes sense!

Reply:
It make sense from information we give you in this challenge, but ignoring those booleans means you will discard combinations like
`shared = True` and `curated = True` with `type = Authored`.  
But no big deal in this case 👍

Reply:
Yes, given the types in this challenge, it doesn't make sense to think about combinations of `shared+curated+user`.
(But in our codebase this has indeed evolved into three separate fields. :sweat_smile: )